code401challenges/Data-Structures/src/test/java/Data/Structures/LinkedList/LinkedListTest.java

```diff
@@ -0,0 +1,18 @@
+package Data.Structures.LinkedList;
+
+import org.junit.Test;
+import org.w3c.dom.Node;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+
+  //linked list
+  @Test
+  public void testCreateEmptyLinkedList(){
+   LinkedList test = new LinkedListTest();
```

Comment:
Here you are creating a new "LinkedListTest" you should be creating a new LinkedList